pkg/monitor/dynkeepalived.go

```diff
@@ -374,7 +340,20 @@ func KeepalivedWatch(kubeconfigPath, clusterConfigPath, templatePath, cfgPath st
 				}).Debug("EnableUnicast != enableUnicast from cfg file, update EnableUnicast value")
 				newConfig.EnableUnicast = curEnableUnicast
 			}
-			updateUnicastConfig(kubeconfigPath, &newConfig, appliedConfig)
+			err = updateUnicastConfig(kubeconfigPath, &newConfig, appliedConfig)
+			if err != nil {
+				err = handleUpdateError(conn)
+				if err != nil {
+					return err
+				}
+				time.Sleep(interval)
+				continue
+			}
+			err = ensureRunning(conn)
```

Comment:
I feel a bit uncomfortable with the fact that a start message will be sent every interval to the keepalived container in all nodes other than bootstrap even when everything is OK ( that would be the case 99.99% of the time)

Do you think there's a way to optimize this ? maybe we can start with sending the command only upon a change and then optimize it if needed ? 
I think, if we want to add this PR we should try to optimize the 


start message will be sent with

Reply:
We could, but then we have to keep track of the process state in the monitor too. Right now the state is managed entirely on the haproxy container side so there's no chance of getting out of sync. Since the command is sent over a local socket, the only meaningful load to come out of it is the pgrep that happens to check if keepalived is already running. I don't think that's going to be significant.

We might be able to optimize the other side even more to reuse the results of the liveness probe instead of looking for the process, if that would make this more acceptable.